ash/src/vk/extensions.rs

```diff
@@ -4481,7 +4481,7 @@ impl PipelineCreateFlags {
 #[doc = "Generated from 'VK_KHR_dynamic_rendering'"]
 impl PipelineCreateFlags {
     #[deprecated = "Backwards-compatible alias containing a typo"]
-    pub const RASTERIZATION_STATE_FRAGMENT_SHADING_RATE_ATTACHMENT_KHR: Self =
+    pub const PIPELINE_RASTERIZATION_STATE_CREATE_FRAGMENT_SHADING_RATE_ATTACHMENT_KHR: Self =
```

Comment:
Why is the name changing as well?

Reply:
Because I came up with this "more correct" prefix and added explicit fallback names back when these were the only enums in the source (before being turned into fallback-aliases), instead of throwing them in `is_enum_variant_with_typo` and calling it a day.